radicle-httpd/src/api.rs

```diff
@@ -51,8 +51,29 @@ impl Context {
         let doc = repo.identity_of(self.profile.id())?;
         let payload = doc.project()?;
         let delegates = doc.delegates;
-        let issues = (Issues::open(self.profile.public_key, &repo)?).count()?;
-        let patches = (Patches::open(self.profile.public_key, &repo)?).count()?;
+        let issues = Issues::open(self.profile.public_key, &repo)?.all()?;
+        let issues = issues.filter_map(|s| s.ok()).fold(
+            Issues::default_state_count(),
+            |mut state, (_, i, _)| {
+                match i.state() {
+                    issue::State::Open => state.open += 1,
+                    issue::State::Closed { .. } => state.closed += 1,
+                }
+                state
+            },
+        );
```

Comment:
I actually meant moving this logic into a method. The `default` was fine :smile:

Reply:
ok nice, great idea.
Done in 54d489b